pages/reference-smart-account/owners/removeOwner.mdx

```diff
@@ -0,0 +1,97 @@
+import { Tabs, Callout } from 'nextra/components'
+
+# `removeOwner`
+
+## Overview
+
+Removes the owner `owner` from the Safe and updates the threshold to `_threshold`.
+
+<Callout type='warning'>This can only be done via a Safe transaction.</Callout>
```

Comment:
This could be replaced in the other instances of this warning as well.
```suggestion
<Callout type='warning'>This action can only be done via a Safe transaction.</Callout>
```